hudi-common/src/test/java/org/apache/hudi/metadata/TestHoodieTableMetadataUtil.java

```diff
@@ -281,37 +283,27 @@ void testFilterNestedLogicalTimestampColumn() {
   @Test
   void testIsTimestampMillisField() {
     // Test timestamp-millis
-    Schema timestampMillisSchema = Schema.create(Schema.Type.LONG);
-    LogicalTypes.timestampMillis().addToSchema(timestampMillisSchema);
+    HoodieSchema timestampMillisSchema = HoodieSchema.createTimestampMillis();
     assertTrue(HoodieTableMetadataUtil.isTimestampMillisField(timestampMillisSchema),
         "Should return true for timestamp-millis");
 
-    // Test local-timestamp-millis
-    Schema localTimestampMillisSchema = Schema.create(Schema.Type.LONG);
-    LogicalTypes.localTimestampMillis().addToSchema(localTimestampMillisSchema);
-    assertTrue(HoodieTableMetadataUtil.isTimestampMillisField(localTimestampMillisSchema),
-        "Should return true for local-timestamp-millis");
```

Comment:
Should this test be preserved?

Reply:
CMIIW, we do not have a a local timestamp millis schema, which is why i removed this.